protocol/triple/triple-tool/internal/proto/greettriple/greet.triple.go

```diff
@@ -0,0 +1,386 @@
+// exposed at runtime as Spec.Procedure and as the final two segments of the HTTP route.
+//
+// Note that these are different from the fully-qualified method names used by
+// google.golang.org/protobuf/reflect/protoreflect. To convert from these constants to
+// reflection-formatted method names, remove the leading slash and convert the remaining slash to a
+// period.
+const (
+	// GreetServiceGreetProcedure is the fully-qualified name of the GreetService's Greet RPC.
+	GreetServiceGreetProcedure = "/greet.GreetService/Greet"
+	// GreetServiceGreetStreamProcedure is the fully-qualified name of the GreetService's GreetStream RPC.
+	GreetServiceGreetStreamProcedure = "/greet.GreetService/GreetStream"
+	// GreetServiceGreetClientStreamProcedure is the fully-qualified name of the GreetService's GreetClientStream RPC.
+	GreetServiceGreetClientStreamProcedure = "/greet.GreetService/GreetClientStream"
+	// GreetServiceGreetServerStreamProcedure is the fully-qualified name of the GreetService's GreetServerStream RPC.
+	GreetServiceGreetServerStreamProcedure = "/greet.GreetService/GreetServerStream"
+)
+
+var (
+	_ GreetService = (*GreetServiceImpl)(nil)
+
+	_ GreetService_GreetStreamClient       = (*GreetServiceGreetStreamClient)(nil)
+	_ GreetService_GreetClientStreamClient = (*GreetServiceGreetClientStreamClient)(nil)
+	_ GreetService_GreetServerStreamClient = (*GreetServiceGreetServerStreamClient)(nil)
+
+	_ GreetService_GreetStreamServer       = (*GreetServiceGreetStreamServer)(nil)
+	_ GreetService_GreetClientStreamServer = (*GreetServiceGreetClientStreamServer)(nil)
+	_ GreetService_GreetServerStreamServer = (*GreetServiceGreetServerStreamServer)(nil)
+)
+
+//GreetService is a client for the greet.GreetService service.
+type GreetService interface {
+	Greet(ctx context.Context, req *proto.GreetRequest, opt ...client.CallOption) (*proto.GreetResponse, error)
+
+	GreetStream(ctx context.Context, opt ...client.CallOption) (GreetService_GreetStreamClient, error)
+
+	GreetClientStream(ctx context.Context, opt ...client.CallOption) (GreetService_GreetClientStreamClient, error)
+
+	GreetServerStream(ctx context.Context, req *proto.GreetServerStreamRequest, opt ...client.CallOption) (GreetService_GreetServerStreamClient, error)
+}
+
+// NewGreetService constructs a client for the greet.GreetService service. By default, it uses
+// the Connect protocol with the binary Protobuf Codec, asks for gzipped responses, and sends
+// uncompressed requests. To use the gRPC or gRPC-Web protocols, supply the connect.WithGRPC() or
+// connect.WithGRPCWeb() options.
+//
+// The URL supplied here should be the base URL for the Triple server (for example,
+// http://api.acme.com or https://acme.com/grpc).
+func NewGreetService(cli *client.Client) (GreetService, error) {
+	if err := cli.Init(&GreetService_ClientInfo); err != nil {
+		return nil, err
+	}
+	return &GreetServiceImpl{
+		cli: cli,
+	}, nil
+}
+
+func SetConsumerService(srv common.RPCService) {
```

Comment:
Since we have not figured out how to configure with dubbogo.yml. We should remove this statement.